config/initializers/flipper.rb

```diff
@@ -38,6 +38,24 @@
   actor.respond_to?(:created_at?) && actor.created_at >= "2025-08-11".to_time
 end
 
+# This is group is to be used to turn on variant tags for enterprises with inventory enabled but
+# are not using it
+Flipper.register(:old_enterprise_with_no_inventory) do |actor|
+  # This group applies to enterprises only, so we return false if the actor is not an Enterprise
+  next false unless actor.actor.instance_of? Enterprise
+
+  enterprise_with_variant_override = Enterprise
+    .where(id: VariantOverride.joins(:hub).select(:hub_id))
+    .where("created_at < ?", "2025-08-11")
+    .distinct
+  enterprise_with_no_variant_override = Enterprise
+    .where.not(id: enterprise_with_variant_override)
+    .where("created_at < ?", "2025-08-11")
+    .pluck(:id)
+
+  enterprise_with_no_variant_override.include?(actor.id)
```

Comment:
I can see how these queries probably get combined and only executed at once, with simply a list of IDs returned from the DB. But it seems unnecessarily complicated, and will probably result in a large list of IDs, when we only need one.

Could there be a simpler way? Or at least, I think we could change the `pluck` to something like `exist?(actor.id)`